frontend/src/pages/meditation/components/meditation-list/meditation-entry/meditation-entry.tsx

```diff
@@ -2,11 +2,14 @@ import meditationListPlaceholder from '#assets/img/meditation-list-placeholder.j
 import { Icon, Link } from '#libs/components/components.js';
 import { IconColor } from '#libs/enums/enums.js';
 
-import { type MeditationEntry } from '../../libs/types/types.js';
 import styles from './styles.module.scss';
 
 type Properties = {
-  meditationEntry: MeditationEntry;
+  meditationEntry: {
```

Comment:
we should pass here type that we will receive from backend

Reply:
@Liza-Veis At now I have these properties: 

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/f94b82b0-e12f-4e8a-ae55-f84bf5d43057)

However from the backend I'm receiving this:

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/b7f0ebe9-c866-4654-8f63-968d07205ffb)

Where should I get duration (I can't get it from the reference, because meditation player isn't implemented) and title (basically topic name, that I setted to null)?
And also topics aren't unque, so what should be used as `key` in loop?

Reply:
@Liza-Veis At now meditation entry has that look:

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/5c668885-f3e0-4a4d-9331-c0951db822eb)

I can't already implement meditation player, because there's no [meditation timer](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/253), therefore I won't have audio reference. So how should I get `duration`?
Also I don't have `title`, because all topic names are null at now. Even though topic nanes aren't unique, so what should I use as loop `key`?

Reply:
we won't have medition topics screen, they will be like tags or label. The items that we see in the sidebar - meditions, relaxing sounds etc. are not meditation topics. They are predefined screens on the frontend. We need to adjust already existed logic on frontend to work with our backend. We won't have duration field.

Reply:
Done, I mocked duration